CONTRIBUTING.md

```diff
@@ -0,0 +1,38 @@
+## How to contribute to HedgeDoc react-client
+
+Thanks for your interest in contributing. Here are some common scenarios for what you may want to contribute to.
+
+#### Do you have questions about the project?
+
+* Feel free to join our [Matrix Support Channel][matrix-support] or post your question on our [Discourse][discourse].
```

Comment:
I would recommend to put discourse first as answers there will allow to be found by search engines making everyone's life easier.

Reply:
done